<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="Test Runner 2(failed)">
    <classes>
      <class name="Practice.CreateVtigerScenarioWithDDT_GUTest">
        <methods>
          <include name="BeforeSuiteConfiguration"/>
          <include name="BeforeMethodConfiguration"/>
          <include name="AfterSuiteConfiguration"/>
          <include name="AfterClassConfiguration"/>
          <include name="BeforeClassConfig"/>
          <include name="cteateVtigerScenarioTest"/>
          <include name="AfterMethodCongiguration"/>
        </methods>
      </class> <!-- Practice.CreateVtigerScenarioWithDDT_GUTest -->
    </classes>
  </test> <!-- Test Runner 2(failed) -->
  <test thread-count="5" parallel="tests" name="Test Runner 1(failed)">
    <classes>
      <class name="Practice.CreateVtigerScenarioWithDDT_GUTest">
        <methods>
          <include name="BeforeSuiteConfiguration"/>
          <include name="BeforeMethodConfiguration"/>
          <include name="AfterClassConfiguration"/>
          <include name="AfterSuiteConfiguration"/>
          <include name="BeforeClassConfig"/>
          <include name="AfterMethodCongiguration"/>
          <include name="cteateVtigerScenarioTest"/>
        </methods>
      </class> <!-- Practice.CreateVtigerScenarioWithDDT_GUTest -->
    </classes>
  </test> <!-- Test Runner 1(failed) -->
</suite> <!-- Failed suite [Suite] -->
